tt_eager/tt_dnn/op_library/prod/prod_op_all.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+#include "tensor/tensor.hpp"
+#include "tt_dnn/op_library/run_operation.hpp"
+
+namespace tt {
+
+namespace operations{
+
+namespace primary{
+
+using namespace constants;
+using namespace tt_metal;
```

Comment:
@arakhmati we don't want using namespace in headers right?

Reply:
@VirdhatchaniKN you should remove the using namespace from this header file. You can keep them for cpp files though.